core/src/test/java/com/linecorp/armeria/client/proxy/ProxyClientIntegrationTest.java

```diff
@@ -118,7 +131,7 @@
     static ServerExtension backendServer = new ServerExtension() {
         @Override
         protected void configure(ServerBuilder sb) throws Exception {
-            sb.port(0, SessionProtocol.HTTP);
+            sb.port(0, HTTP);
```

Comment:
IIRC we don't need this at all. If no port is specified, it automatically adds this for you.